sbatch_sh_files/verify_barlowTwins.sh

```diff
@@ -0,0 +1,35 @@
+#!/usr/bin/env bash
+####SBATCH --array=0-47%16
+#SBATCH --partition=long
+#SBATCH --gres=gpu:a100:1
+#SBATCH --reservation=DGXA100
+#SBATCH --mem=16GB
+#SBATCH --time=4:00:00
+#SBATCH --cpus-per-gpu=4
+#SBATCH --output=sbatch_out/exp_track_alpha_barlowtwins.out
+#SBATCH --error=sbatch_err/exp_track_alpha_barlowtwins.err
+#SBATCH --job-name=exp_track_alpha_barlowtwins
+
+. /etc/profile
+module load anaconda/3
+conda activate ffcv
+
+lambd=0.00397897
+pdim=3072
+dataset='cifar10'
+if [ $dataset = 'stl10' ]
+then
+    batch_size=256
+else
+    batch_size=512
+fi
+
+checkpt_dir=$SCRATCH/fastssl/checkpoints
+
```

Comment:
can we add a line for each script lines below for readability/documentation?

Reply:
Yes, I have done some of that in the other PR. I will add more details to improve readability.